### PR TITLE
Update building.md

### PR DESCRIPTION
**Pull request by HexaOneOfficial**
fix some typo's and other small things

### CODE REVIEW

File: building.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ $ git clone --recursive https://github.com/skiftOS/skift
 $ cd skift
 ```
 
-Or if you already clone this repo without `--recursive` do:
+Or if you already cloned this repo without `--recursive` do:
```
Comment (Keyboard-Slayer) [inline building.md:56]: `if you have already` cloned will be better I think

File: building.md [hunk 2]
```diff
@@ -100,9 +100,9 @@ From the root of this repo do:
 $ make all
 ```
 
-This command will build all the component of the project and generate an ISO bootable in QEMU or VirtualBox.
+This command will build all the components of the project and generate an bootable ISO in QEMU or VirtualBox.
```
Comment (Keyboard-Slayer) [inline building.md:103]: `a bootable ISO`

**Review by Keyboard-Slayer — CHANGES_REQUESTED** [review]

**Comment by caelwithcats** [comment]
```
This branch has no conflicts with the base branch
```
so why did you add the `stale` tag?

**Comment by sleepy-monax** [comment]
It's stale because the discussion are marked as resolved but there is not commit, wich fix the issues brought by the reviewer

**Comment by HexaOneOfficial** [comment]
sure close it than?

**Comment by caelwithcats** [comment]
@HexaOneOfficial just add more commits to this PR

**Pull request closed without merging**